packages/tasks/src/tasks/audio-to-audio/inference.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+/**
+ * Inputs for Audio to Audio inference
+ */
+export interface AudioToAudioInput {
+	/**
+	 * The input audio data
+	 */
+	inputs: unknown;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: {
+		[key: string]: unknown;
+	};
+	[property: string]: unknown;
+}
+export type AudioToAudioOutput = AudioToAudioOutputElement[];
+/**
+ * Outputs of inference for the Audio To Audio task
+ *
+ * A generated audio file with its label.
+ */
+export interface AudioToAudioOutputElement {
```

Comment:
FYI here is the returned values from community API https://github.com/huggingface/api-inference-community/blob/main/docker_images/speechbrain/app/pipelines/audio_to_audio.py#L37-L44